card/db/schema.rb

```diff
@@ -38,7 +38,7 @@
   create_table "card_changes", force: :cascade do |t|
     t.integer "card_action_id", limit: 4
     t.integer "field",          limit: 4
-    t.text    "value",          limit: 65535
+    t.text    "value",          limit: 16777215
```

Comment:
No, we don't really know how/why.  my guess is that defaults for autogeneration of schema.rb changed with rails 4. I haven't looked into that, though, so.